scanomatic/generics/abstract_model_factory.py

```diff
@@ -145,43 +146,40 @@ class AbstractModelFactory:
     MODEL = Model
 
     _LOGGER = None
-    _SUB_FACTORIES: dict[Model, "AbstractModelFactory"] = {}
+    _SUB_FACTORIES: dict[Type[Model], "AbstractModelFactory"] = {}
```

Comment:
ah va det detta som va fel på förra PRn att det skule vara type of Model och inte instans av Model?

Reply:
nu kanske inte andra klasser som ärver av den här behöver definiera om sig? Också, stämmer det verkligen att det e en instans av AbstractModelFactory? är det inte också typen?

Reply:
Det är en bra fråga... Ska kolla.

Reply:
Ja, jag tycker det ser ut så:
```python
    _SUB_FACTORIES = {
        compile_project_model.CompileImageModel: CompileImageFactory,
    }
```

Reply:
jag tror dom där som heter Factory är "statiska" klasser

Reply:
Så ingen ändring behövs?

Reply:
jo borde det inte va `dict[Type[Model], Type["AbstractModelFactory"]]`?

Reply:
fixat